examples/discovery-search-app/cypress/fixtures/query/passageResults.json

```
@@ -24,19 +24,19 @@
         "file_type": "pdf"
       },
       "text": [
-        "This is a document.\nThis result multiple passages, but you should only be able to see the first one. IBM if you can see this passage, something probably borked"
+        "This is a document.\nThis result multiple passages, so we'll display them in the order that they're returned. IBM We now display all passages that are returned for each document."
```

These sentences are hurting me.
```suggestion
        "This is a document.\nThis result has multiple passages, so we'll display them in the order that they're returned. IBM. We can display multiple passages that are returned for each document."
```

Also removed "all" since it depends on the setting.

---

I'm fine with the language change, but the component will actually display all of the passages returned (but the number returned will be limited by `max_per_document`).

---

Right, that's what I was getting it: it will be limited by `max_per_document`, so not technically "all".